client/src/controllers/KeyboardController.test.js

```diff
@@ -3,6 +3,12 @@ import Mousetrap from 'mousetrap';
 
 import { KeyboardController } from './KeyboardController';
 
+jest.mock('../config/wagtailConfig', () => ({
+  WAGTAIL_CONFIG: {
+    KEYBOARD_SHORTCUT_PREFERENCE: true,
```

Comment:
It might be better to add the 'default' (true) to the core stubs instead.

`client/tests/stubs.js`